src/components/NavigatorSearch/modes/commands/actions.test.ts

```diff
@@ -0,0 +1,36 @@
+import { useIntl } from "react-intl";
+
+import getCommandModeActions from "./actions";
+
+jest.mock("react-intl", () => {
+  const reactIntl = jest.requireActual("react-intl");
+  const intl = reactIntl.createIntl({
+    locale: "en",
+  });
+
+  return {
+    ...reactIntl,
+    useIntl: () => intl,
+  };
```

Comment:
suggestion: using actual react-intl does not have sense because it doesn't work without context

Reply:
Nope it doesn't, it works in your case because you mocked useIntl, but you could do this like that:
```suggestion
jest.mock("react-intl", () => ({
  useIntl: jest.fn(() => ({
    formatMessage: jest.fn(x => x.defaultMessage),
  })),
}));
```

Reply:
I meant that it does work 😄

Reply:
@Cloud11PL test should mock dependencies, not mock and provide their actual implementation. You are not testing react-intl here, you test only your code and the only you care about is that your code calls intl with proper arguments.